### PR TITLE
Alpha 2 preparations

### PR DESCRIPTION
**Pull request by Hornwitser**
Lots of tiny things in preparation for alpha.2.  See commits for detailed descriptions but resolves mostly around docs fixes, bug fix in generate-token command, lint fixes, some moving in lib to avoid warnings and imports not available in web build, and responsive design fixes for the web UI.

### CODE REVIEW

File: packages/lib/factorio/patch.js [hunk 1]
```diff
@@ -152,7 +152,7 @@ function reorderDependencies(modules) {
 
 		// No unmet dependencies, record and continue
 		present.set(module.name, module.version);
-		index++;
+		index += 1;
```
Comment (psihius) [inline packages/lib/factorio/patch.js:155]: Is there like a lint rule or something that mandate changing it from increment to this form?

File: packages/lib/link/connectors.js [hunk 2]
```diff
@@ -102,12 +102,14 @@ class WebSocketBaseConnector extends events.EventEmitter {
 		if (!["handshake", "connected", "closing"].includes(this._state)) {
 			throw new libErrors.SessionLost("No session");
 		}
-		let message = { seq: this._seq, type, data };
+		let seq = this._seq;
```
Comment (psihius) [inline packages/lib/link/connectors.js:105]: I'm curious - race conditions or just making code more readable and understandable? :)

**Review by Danielv123 — APPROVED** [review]
Looks good, I love how clean the tests are

File: packages/lib/link/connectors.js (hunk 2 above)
Comment (Danielv123) [reply packages/lib/link/connectors.js:105]: Readability - he increments this._seq, previously it was done in the return statement further down which was unclear.

File: packages/lib/factorio/patch.js (hunk 1 above)
Comment (Danielv123) [reply packages/lib/factorio/patch.js:155]: There was a warn for it, he made it an error instead. See .eslintrc.js https://github.com/clusterio/factorioClusterio/pull/337/commits/5337280eec4426873e11e6aedbc6602ecc904a94

**Review by psihius — APPROVED** [review]

**Pull request merged**